### DISCUSSION TITLE
v1.1.1 for tvOS?

### DISCUSSION THREAD

**Discussion by rwjack** [jellyfin/Swiftfin#1099] [Questions] [answered]
Hey, does anyone know if tvOS is also getting this version? I see the updated app on my phone, but not on the TV.

I thought that once "pushed" to the app store, all versions are deployed, but I never developed for the app store so I wouldn't know.

**Comment by LePips** [comment, marked as answer]
I apologize, but tvOS is not shippable in its current state and does not have this 1.1 update. For some technical clarity, the iOS and tvOS apps are essentially two separate coding projects, so just pushing one does not mean anything for the other. Each project will receive their respective updates when they are ready. I won't go into deep details on versioning. 

Please join the Jellyfin iOS development channel to see previous comments for the current state of tvOS.
- https://jellyfin.org/contact

**Reply by rwjack** [reply]
Ahh okay, that makes sense. Really liking the iOS quality of life improvements so far, have no doubt it will be as good for tvOS!!!

**Reply by rwjack** [reply]
@LePips BTW, while I have you here, a slight modification to the readme would be cool:

From:
`<a href="https://matrix.to/#/+jellyfin:matrix.org">`
To:
`<a href="https://matrix.to/#/#jellyfin:matrix.org">`

The + sign denotes a Matrix Group, which according to the devs is an old standard that was not implemented and replaced by Matrix Spaces, so we should likely be directing users to the #jellyfin room. (Same as on the [jellyfin repo](https://github.com/jellyfin/jellyfin/blob/f39a17de140e959cc3c9548f2536e3ca59c96382/README.md?plain=1#L29C29-L29C48))

**Comment by muggajoe** [comment]
Makes sense. Recommend, if possible, changing the list of supported platforms for Swiftfin in the Apple App Store. Currently it purports to support Apple TV. 
![Screenshot 2024-08-31 at 2 12 41 PM jpeg](https://github.com/user-attachments/assets/adf17257-5c94-4f14-95c4-d3cd46f83617)

**Comment by crystalcommunication** [comment]
Swiftfin does support the Apple TV. The App Store doesn't care about the nuanced differences between the app for each platform, as long as there is _a version_ that can be installed on the Apple TV.

While I eagerly await the addition of new features for the tvOS version, I would like to express my gratitude that a decent native Jellyfin client exists for the Apple TV at all, the device would be relatively useless to me without it.

**Reply by RedNo7** [reply]
I get you, but the tvOS version is useless for me; everything either stutters or just does't play at all - no matter the settings.  Given 99% of my entertainment content is on my TV (not iOS devices) this is the single reason I still use Plex so really looking forward to an update.

Edit: ...to be clear, I also appreciate that they are working on it!

**Comment by ryanlbates01** [comment]
Hi all.

just wanted to know if there was any rough timeline for a tvOS update? 

I’ve come to realise the current tvOS version is more buggy than I first realised. I’ve now confirmed there are audio-sync issues with the current version which now I’ve realised it is making it unusable. I’ve since had to move over to infuse player. 

the iOS version works great and would love to be able to have the same experience on my appleTV as this is how I view my content. 

Appreciate all the work you guys put in.

regards

Ryan

**Reply by jamesstorm** [reply]
Agreed. Kudos to the folks doing all the work. A tvOS app on par with the iOS app is the last thing preventing me from dumping Plex.
